### PR TITLE
refactor homepage - 1/N

### PR DESCRIPTION
**Pull request by fusionstrings**

### CODE REVIEW

File: server.jsx [hunk 1]
```diff
@@ -3,22 +3,47 @@
...
 import { Package } from "./lib/package.js";
-import { FeaturedPackages } from "./lib/featured-packages.js";
+import { Home } from "./lib/home.js";
 import { pageServingHeaders, renderMarkdownContent } from "./utils.js";
-import { featuredPackages } from "./lib/featured-packages-list.js";
+import { FEATURED_PACKAGES } from "./lib/featured-packages-list.js";
 
 const staticResources = {
   "/style.css": { path: "./style.css", contentType: "text/css; charset=utf-8" },
+  "/package/style.css": {
```
Comment (JayaKrishnaNamburu) [inline server.jsx:12]: Was checking to solve this, can we regex the URL and pick the last filename and pass to this map ?

**Comment by JayaKrishnaNamburu** [comment]
Might not be directly related to this PR, but we should make few contents of this dynamic.
https://github.com/jspm/jspm-packages/blob/main/src/shell.html#L12

This loads `prism.css` in every page even when not used like `home` page.

**Comment by fusionstrings** [comment]
> Might not be directly related to this PR, but we should make few contents of this dynamic. https://github.com/jspm/jspm-packages/blob/main/src/shell.html#L12
> 
> This loads `prism.css` in every page even when not used like `home` page.
> 
> Or we can move inside the component. So, it get's loaded while hydrating. Anyways these changes won't apply in SSR. So, good to leave for hydration

must be fixed now

File: server.jsx (hunk 1 above)
Comment (fusionstrings) [reply server.jsx]: thanks for reviewing. must be fixed now

**Pull request merged**